processor/processor.go

```diff
@@ -731,13 +731,15 @@ func (proc *Handle) backendConfigSubscriber(ctx context.Context) {
 				}
 			}
 			workspaceLibrariesMap[workspaceID] = wConfig.Libraries
+			eventAuditEnabled = wConfig.Settings.EventAuditEnabled
```

Comment:
You would override the value in this loop as it is running for multiple workspaces right ?